apis/github

```diff
@@ -38,17 +33,32 @@ Auth.getAll = function()
 	end
 	return authTable
 end
+local Auth = {}
+Auth.__index = Auth
+Auth.new = function(type, user, token)
+	return setmetatable({type=type, user=user, token=token}, Auth)
+end
 Auth.get = function(user)
-	local authTable = Auth.getAll()
-	return authTable[user]
+	local authTable = getAuthTable()
+	local auth = authTable[user]
+	if auth then
+		auth = Auth.new(auth.type, auth.user, auth.token)
+	end
+	return auth
+end
+Auth.checkToken = function(self)
+	local url = 'https://api.github.com/user'
+	local headers = generateAuthHeaders(self)
+	local req = http.get(url, headers)
```

Comment:
Wait no it shouldn't I'm relying on a request and a request status of 200 getAPI doesn't return a request.